pipeline_dp/budget_accounting.py

```diff
@@ -196,11 +246,29 @@ def _normalise_mechanism_weights(self):
         for mechanism in self.mechanisms:
             mechanism.weight *= normalisation_factor
 
+    def _compute_budget_for_aggregation_scope(self):
+        n_aggregations = self.accountant._n_aggregations
+        weights = self.accountant._aggregations_weights
+        if n_aggregations is not None:
+            budget_ratio = 1 / n_aggregations
```

Comment:
need to add a check for if `_aggregations` is 0

Reply:
It's checked in the contrustor, that it should be > 0